README.md

```diff
@@ -1 +1,116 @@
-# intro-to-tests-activity
+# Intro To Tests Activity 
+This repository is part of the class activity for Intro to Tests in Unit 1.
+
+## Learning Goals
+- Practice identifying test cases
+- Practice writing tests with pytest
+
+### Part 1: One-Time Setup
+
+Follow these directions once when you start working on this activity **during roundtables**:
+
+1. Navigate to your projects folder named `projects`
+
+```bash
+$ cd ~/Developer/projects
+```
+
+1. In Github click on the green "Code" button in Github and then copy the URL. This will allow you to download a copy of this project into your projects folder. This command makes a new folder called `intro-to-tests-activity`, and then puts the project into this new folder. 
+
+![Cloning a repository in Github](images/cloning_a_repo.png)
+
+After copying the URL, run the following command in your terminal:
+
+```
+$ git clone <paste your copied URL here>
+```
+
+Use `ls` to confirm there's a new project folder
```

Comment:
I can't submit it as a suggestion because of the code block, but I had a little confusion with the `clone` command description being up by the `cd` command, how does this flow?

> 1. In Github click on the green "Code" button in Github and then copy the URL. 
> 
> ![Cloning a repository in Github](images/cloning_a_repo.png)
> 
> After copying the URL, run the following command in your terminal:
> 
> ```py
> $ git clone <paste your copied URL here>
> ```
> This will allow you to download a copy of this project into your projects folder. This command makes a new folder called `intro-to-tests-activity`, and then puts the project into this new folder. 
> 
> Use `ls` to confirm there's a new project folder
>